modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1827,6 +1827,7 @@ up    <string name="player_sleep_in_one_chapter">Sleeping in 1 chapter</string>
     <string name="onboarding_plus_terms_and_conditions">Terms and Conditions</string>
     <string name="onboarding_plus_start_free_trial_and_subscribe">Start free trial &amp; subscribe</string>
     <string name="onboarding_plus_subscribe">Subscribe to Pocket&#160;Casts Plus</string>
+    <string name="onboarding_subscribe_to_subscribe">Subscribe to Plus</string>
```

Comment:
Is this a typo? Shouldn't it be `onboarding_subscribe_to_plus`?

Reply:
Yes! hehe Nice catch [ec52d56](https://github.com/Automattic/pocket-casts-android/pull/2689/commits/ec52d567211f8fd3904a7b5cf6ed6a2af529019e)